pkgs/servers/dynomite/default.nix

```diff
@@ -0,0 +1,26 @@
+{ fetchFromGitHub, stdenv, autoreconfHook, libressl, git }:
+stdenv.mkDerivation rec {
+  pname = "dynomite";
+  version = "0.6.15";
+
+  src = fetchFromGitHub {
+    owner = "Scriptkiddi";
+    repo = "dynomite";
+    rev = "v${version}";
+    sha256 = "1m2w7xfcapjx32265cnvysk4c03ns206bsg2l153vhb8r5lqn40c";
+  };
+
+  buildInputs = [ libressl ];
+  nativeBuildInputs = [autoreconfHook git ];
```

Comment:
Please be consistent with spacing and add an additional space -> `nativeBuildInputs = [ autoreconfHook git ];`